neptune/internal/backends/hosted_neptune_backend.py

```diff
@@ -121,6 +127,69 @@ def execute_operations(self, operations: List[Operation]) -> None:
     def get(self, _uuid: uuid.UUID, path: List[str]) -> Value:
         pass
 
+    @with_api_exceptions_handler
+    def send_hardware_metric_reports(
+            self,
+            experiment_uuid: UUID,
+            metrics: List[Metric],
+            metric_reports: List[MetricReport]) -> None:
+        verify_type("experiment_uuid", experiment_uuid, uuid.UUID)
+        verify_type("metrics", metrics, list)
+        verify_type("metric_reports", metric_reports, list)
+
+        SystemMetricValues = self.backend_client.get_model('SystemMetricValues')
```

Comment:
This locks us out from changing the name of the model later in backend. Do we want that? what are we getting in return ("compile" time - wise)?

Seems that `verify_type` would assert correct structure and then we can send dicts to bravado api calls